utility/utils.py

```diff
@@ -754,3 +755,124 @@ def download_openshift_installer(version=defaults.INSTALLER_VERSION):
         run_cmd(f"tar xzvf {tarball}")
 
     return installer_filename
+
+
+def generate_yaml_from_template(file_, **kwargs):
+    """
+    Generate dictionary from yaml file
+
+    Args:
+        file_ (str): Template Yaml file path
+
+    Keyword Args:
+        All jinja2 attributes
+
+    Returns:
+        dict: Generated from template file
+
+    Examples:
+        generate_yaml_from_template(file_='path/to/file/name', pv_data_dict')
+    """
+    with open(file_, 'r') as stream:
+        data = stream.read()
+    template = jinja2.Template(data)
+    out = template.render(**kwargs)
+    return yaml.safe_load(out)
+
+
+def delete_file(file_name):
+    """
+    Delete file_name
+
+    Args:
+        file_name (str): Path to the file you want to delete
+    """
+    os.remove(file_name)
+
+
+class TimeoutExpiredError(Exception):
```

Comment:
Consider move to [exceptions](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs/exceptions.py)

Reply:
Done